pkg/operator/ceph/cluster/mon/util_test.go

```diff
@@ -0,0 +1,108 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package mon
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/rook/rook/pkg/clusterd"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/fake"
+)
+
+const (
+	monValueFrom = "monitors"
+)
+
+func createFakeSecret(name, ns string, clientset *fake.Clientset) error {
+	data := make(map[string][]byte, 1)
+	data[monValueFrom] = []byte("mon1:6790,mon2:6790")
```

Comment:
Why not just initialize the map directly with the values?:
```
data := map[string][]byte{
    "monitors": []byte("mon1:...."),
}
```

Even though using `make(...., 1)` might save a bit of allocated memory ;)